text/0101-new-scale-control-model-based-on-IO-size.md

```diff
@@ -0,0 +1,99 @@
+# Feedback-based scale control model
+- RFC PR: https://github.com/tikv/rfcs/pull/101
+- Tracking Issue: https://github.com/tikv/pd/issues/5467
+## Summary
+Each store has a separate sending window to limit the number of snapshot tasks. It will reduce the waiting time and balance sending traffic to multiple stores at the same time. 
+## Motivation
+When the difference between the different region's size becomes bigger. It can bring following costs:
+- There is a certain relationship between the speed and size of replica migration. By limiting the number of operators to control the replica speed. It cannot adapt to different region sizes, and cannot associated with IO .
+- The store limit only controls the storage of the source and target instance. In fact, the speed of the operator is mainly affected by the sender and the receiver, so it may cause a serious imbalance in the sending tasks of the individual nodes.
+- The TIKV executes snapshot tasks by FIFO. When the consumption speed of the sender is lower than that of the operator producer, it will take unnecessary wait duration, thereby affecting the timeliness of the operator. So the task will be held longer and other scheduler cannot scheduler this region. 
+Hence, I propose using a new flow mechanism to control the concurrent flow of the sender. It will be helpful to solve the above problems.
```

Comment:
Blank line should be reserved between paragraph. You can see whether it renders correctly by clicked the rich diff button.

Reply:
fix